src/scry/log.cr

```diff
@@ -1,28 +1,8 @@
-require "logger"
+require "log"
 require "./client"
 
 module Scry
   module Log
-    class_property logger : Logger = Logger.new(nil)
-
-    class ClientLogger < Logger
```

Comment:
Warning! The new logger is not configured anymore, the log messages do not go anywhere..

You should have a custom backend that inherits from Log::IOBackend (I'm writing from memory), that writes messages to LSP with a "window/logMessage" event (see a few line below for example)

And then in the initialization code, this backend should be configured to receive all supported log messages.

Reply:
I initially tried inheriting from that class but run into issues. IIRC, the write method was changed to expect an `Entry` parameter. I'll try to make the changes that you suggest. Thank you very much for the feedback.

Reply:
Yes you now take an `Entry` structure that describes the log event, and you extract the fields relevant for you from it.